gensrc/thrift/InternalService.thrift

```diff
@@ -159,6 +159,8 @@ struct TQueryOptions {
   54: optional i32 pipeline_dop;
   // For pipeline query engine
   55: optional TPipelineProfileLevel pipeline_profile_level;
+  // For load parallel request num
+  56: optional i32 load_parallel_request_num;
```

Comment:
```
load_dop
```
dop is standard for the degree of parallel. The query execution already uses this noun. It's better to be consistent.